torch/_torch_docs.py

```diff
@@ -5182,13 +5182,24 @@ def merge_dicts(*dicts):
     >>> torch.quantile(a, q)
     tensor([-0.5446,  0.0700,  0.9214])
 
-.. function:: quantile(input, q, dim=None, keepdim=False, *, out=None) -> Tensor
+.. function:: quantile(input, q, dim=None, interpolation='linear', keepdim=False, *, out=None) -> Tensor
 
-Returns the q-th quantiles of each row of the :attr:`input` tensor along the dimension
-:attr:`dim`, doing a linear interpolation when the q-th quantile lies between two
-data points. By default, :attr:`dim` is ``None`` resulting in the :attr:`input` tensor
+Returns the q-th quantiles of each row of the :attr:`input` tensor
+along the dimension :attr:`dim` based on :attr:`interpolation`.
+When the desired quantile lies between two data points ``i < j``,
+the result is computed based on the :attr:`interpolation` value as described below.
+By default, :attr:`interpolation` is ``linear`` and :attr:`dim` is ``None`` resulting in the :attr:`input` tensor
```

Comment:
This sentence seems like it can be removed, too.

Specifying the default behavior is interesting, but the conflation of the interpolation mode with the flattening behavior of dim here may be confusing (since they're unrelated).